plutus-core-spec/figures/PlutusCoreReduction.tex

```diff
@@ -9,9 +9,9 @@
 \begin{figure*}
     \centering
     \[\begin{array}{lrclr}
-        \textrm{Type Frame} & f  & ::= & \inFunTLeft{\_}{A}                        & \textrm{left arrow}\\
+        \textrm{Type Frame} & f  & ::= & \inConT{tcn}{\_}                          & \textrm{type constant}\\
```

Comment:
Why did we remove `inFixT`? I thought it was agreed that not normalizing inside `(fix a _)` was bad, and I think some of the proofs depend on being able to normalize inside them.